lib/correios/frete/version.rb

```diff
@@ -1,6 +1,6 @@
 # encoding: UTF-8
 module Correios
   module Frete
-    VERSION = "1.10.1"
+    VERSION = "1.10.2"
```

Comment:
Who decides when and how the version bumps are the project maintainers, on the root project branches, not in forks. The release of a new version not necessarily is composed by a merge of only one PR, it could be more than one modification related with different features, fixes, etc. 

It's not the case of this PR, but the rule is the same.

One hypothetical example: if we submit a pull request to Rails, does it make sense we change Rails version in our fork? 😃 

Anyway, thank you for your contribution. 👍